api/ingester/v1/ingester.proto

```diff
@@ -177,6 +177,9 @@ message MergeProfilesPprofRequest {
   // The client starts the stream with a request containing the profile type and the labels.
   SelectProfilesRequest request = 1;
 
+  // Max nodes in the resulting profile.
+  optional int64 max_nodes = 3;
```

Comment:
a naive question perhaps, should this go below profiles so that they are in order?

Reply:
I decided to put fields in this order for consistency with other requests incapsulating `SelectProfilesRequest` and to logically separate options specific to `MergeProfilesPprofRequest` and the profile batch which is the only member we actually stream